src/views/LandingPage/index.js

```diff
@@ -4,6 +4,7 @@ import {toggleLandingPageForm} from "../../redux/actions/applicationActions";
 
 const mapStateToProps = (state) => ({
   formType: state.application.showRegisterFirst ? "register" : "login",
+  hasUser: (state.users && Object.keys(state.users.userInfo).length > 0) ? true : false,
```

Comment:
you definitely don't need to do a tertiary. just do (state.users && Object.keys(state.users.userInfo).length > 0)